util/version/version.go

```diff
@@ -21,22 +21,25 @@ var ErrInvalidSemVer = errors.New("invalid semantic version")
 
 // MustParse - must parse version, if fails - panics
 func MustParse(version string) *types.Version {
-	ver, err := GetVersion(version)
-	if err != nil {
-		panic(err)
+	ver := GetVersion(version)
+	if ver.Invalid != nil {
+		panic(ver.Invalid)
 	}
 	return ver
 }
 
 // GetVersion - parse version
-func GetVersion(version string) (*types.Version, error) {
+func GetVersion(version string) *types.Version {
```

Comment:
would it make sense to have a field on types.Version whether it's a semver tag or not? Maybe we could have different types like:
- VersionTypeSemver
- VersionTypeNonSemver 
- some other version type, maybe build number or whatever  

Using Invalid is a bit weird because apparently we would be using it as a valid thing in a lot of places

Reply:
I agree :) I'll give it a type instead of an error.